polars/polars-lazy/src/frame/parquet.rs

```diff
@@ -40,6 +40,39 @@ impl LazyFrame {
         Ok(lf)
     }
 
+    fn concat_impl(lfs: Vec<LazyFrame>, args: ScanArgsParquet) -> Result<LazyFrame, PolarsError> {
```

Comment:
Extracted the `concat` section into this method because the same code is used in two places.